testing/trino-product-tests-launcher/src/main/resources/docker/trino-product-tests/conf/environment/multinode-apache-ranger/access-control.properties

```diff
@@ -1,4 +1,4 @@
-access-control.name=apache-ranger
+access-control.name=ranger
```

Comment:
There are still `apache-` prefix usages - env, suite and group name for product tests.

Reply:
Yeah .. but those are not user facing .. I am wondering if we want to change all those as well as the filename for the docs or not .. wdyt @ebyhr and @martint ? I just wanted to get master to build for starters..